modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutableTable.java

```diff
@@ -35,11 +33,6 @@ public interface ExecutableTable {
      */
     UpdatableTable updatableTable();
 
-    /**
-     * Requests a current colocation group for this table.
-     */
-    CompletableFuture<ColocationGroup> fetchColocationGroup();
```

Comment:
after this removing there are unused variables in ExecutableTableImpl plz fix them

Reply:
done